From e8a8ff62dfaad1ee42606f1ffd45c7e82288f293 Mon Sep 17 00:00:00 2001
From: Chen Liangjun <b36089@freescale.com>
Date: Fri, 4 Jan 2013 13:27:52 +0800
Subject: [PATCH 380/505] ENGR00238829 ASRC TEST: add init value for ASRC pair
 index

Add init value for pair_index. If the file to be converted is unvalid,
ASRC_RELEASE_PAIR would be called. In this case, the pair_index's init
value is ASRC_PAIR_A, an system hang would happens.

Signed-off-by: Chen Liangjun <b36089@freescale.com>
---
 test/mxc_asrc_test/mxc_asrc_test.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/test/mxc_asrc_test/mxc_asrc_test.c b/test/mxc_asrc_test/mxc_asrc_test.c
index 2ba33a4..2165ca0 100644
--- a/test/mxc_asrc_test/mxc_asrc_test.c
+++ b/test/mxc_asrc_test/mxc_asrc_test.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2008-2012 Freescale Semiconductor, Inc. All rights reserved.
+ * Copyright 2008-2013 Freescale Semiconductor, Inc. All rights reserved.
  */
 
 /*
@@ -445,6 +445,7 @@ int main(int ac, char *av[])
 	inclk = INCLK_NONE;
 	outclk = OUTCLK_ASRCK1_CLK;
 	convert_flag = 0;
+	pair_index = ASRC_UNVALID_PAIR;
 	printf("Hi... \n");
 
 	if (ac < 5) {
-- 
1.8.0

